src/mock/destination-mock.js

```diff
@@ -0,0 +1,20 @@
+import {getRandomArrayElement} from '../utils.js';
+import {CITIES, CITY_DESCRIPTION} from '../constant.js';
+
+const generateDestination = () => {
+  const city = getRandomArrayElement(CITIES);
+
+  return {
+    id: crypto.randomUUID(),
+    name: city,
+    description: CITY_DESCRIPTION,
+    pictures: [
+      {
+        'src': `https://loremflickr.com/248/152?random=${crypto.randomUUID()}`,
```

Comment:
Ключи объекта не следует помещать в строки, оно и так будет строкой
```suggestion
        src: `https://loremflickr.com/248/152?random=${crypto.randomUUID()}`,
```